lib/beaker-pe/install/pe_utils.rb

```diff
@@ -409,7 +409,14 @@ def deploy_frictionless_to_master(host)
               klass = "pe_repo::platform::windows_x86_64"
             end
           else
-            klass = "pe_repo::platform::#{klass}"
+            # Redhat FIPS is an awkward special case we need to check for
+            # where the platform is el-7 but the packaging_platform is
+            # redhat-fips-7:
+            if host['packaging_platform'] == "redhat-fips-7-x86_64"
```

Comment:
Can this be fixed in beaker-hostgenerator so that it provides the correct platform class of redhatfips-7-x86_64 instead of redhat-fips-7-x86_64 so we don't have to correct that here, or is redhat-fips tied into the puppet pipelines somehow?

Reply:
Unfortunately there would also need to be changes in beaker to recognize the platform, which was part of the motivation for using the separate platform/packaging_platform strings in this case. For more context please read the discussion on https://github.com/puppetlabs/beaker-hostgenerator/pull/107. We seem to be going in circles here.